django/utils/html_parser.py

```diff
@@ -4,10 +4,7 @@
 
 current_version = sys.version_info
 
-use_workaround = (
-    (current_version < (2, 7, 3)) or
-    (current_version >= (3, 0) and current_version < (3, 2, 3))
-)
+use_workaround = current_version < (2, 7, 3)
```

Comment:
I don't feel strongly either way in this case. If there were a maintenance overhead here I'd definitely be in favor of enforcing our "we only support the latest minor release" more aggressively.